### DISCUSSION TITLE
Build support for Gravity Forms

### DISCUSSION THREAD

**Discussion by remcotolsma** [pronamic/wp-pronamic-maps#2] [Ideas]
Should we add address autocompletion for Gravity Forms like https://github.com/growella/gravityforms-autocomplete-fields?

For example, once there is a `autocomplete="street-address"` and `autocomplete"postal-code"` attribute?

https://developer.mozilla.org/en-US/docs/Web/HTML/Attributes/autocomplete

![screenshot-1](https://user-images.githubusercontent.com/869674/134190807-9dfd7f59-e20b-4718-b654-048b04f6a333.png)

**Comment by rvdsteege** [comment]
That would be a really nice feature! 👍 However, Gravity Forms does not use any default autocomplete attributes in an address field. If the plugin is triggered by specific autocomplete attributes in the address field, we need to find a way to inform users which attribute names to use.

<img width="309" alt="Schermafbeelding 2021-09-21 om 23 32 05" src="https://user-images.githubusercontent.com/10371164/134250717-8e0cc0b2-29b7-444b-838b-ec85534dd536.png">

**Comment by remcotolsma** [comment]
> However, Gravity Forms does not use any default autocomplete attributes in an address field. 

This is not entirely true, when i add a new Gravity Forms address field and enable the autocomplete feature it does contains the default autocomplete values.
